docs/usage/shoot-operations/shoot_updates.md

```diff
@@ -109,6 +140,127 @@ Changes to `kubeReserved` or `systemReserved` do not trigger a node roll if thei
 
+### InPlace updates
+
+For scenarios where users want to retain the current nodes and avoid their deletion during updates, Gardener provides the option of `in-place` updates. Currently, `in-place` updates are controlled by the `InPlaceNodeUpdates` feature gate.
+
+The machine image version in the `cloudprofile` includes the `inPlaceUpdates` field, which defines the configuration for in-place updates. For an operating system to support in-place updates:
+- The `inPlaceUpdates.supported` field must be set to `true`.
+- The `inPlaceUpdates.minVersionForUpdate` field specifies the minimum version from which an in-place update to the target machine image version can be performed.
+
+```yaml
+machineImages:
+- name: gardenlinux
+  updateStrategy: minor
+  versions:
+  - architectures:
+    - amd64
+    - arm64
+    classification: preview
+    cri:
+    - containerRuntimes:
+      - type: gvisor
+      name: containerd
+    version: 1632.0.0
+    inPlaceUpdates:
+      supported: true
+      minVersionForUpdate: 1630.0.0
+```
+
+The `inPlaceUpdates` field in the Shoot status provides details about in-place updates for the Shoot workers. It includes the `pendingWorkerUpdates` field, which lists the worker pools that are awaiting in-place updates.
+
+#### InPlace Update Triggers
+
+An in-place update of the shoot worker nodes is triggered for rolling update triggers listed under [Rolling Update Triggers](#rolling-update-triggers) except the following:
+* `.spec.provider.workers[].machine.image.name`
+* `.spec.provider.workers[].machine.type`
+* `.spec.provider.workers[].volume.type`
+* `.spec.provider.workers[].volume.size`
+* `.spec.provider.workers[].cri.name`
+* `.spec.systemComponents.nodeLocalDNS.enabled`
+
+#### Validations for In-Place update
+
+When a worker pool is undergoing an in-place update, applying subsequent updates to the same worker pool is restricted.
+If an in-place update fails and nodes are left in a problematic state, user intervention is required to manually fix the nodes. After resolving the issue, users can restart the update by adding the force update annotation `gardener.cloud/operation=force-in-place-update`.
+
+Without this annotation, subsequent updates to the same worker pool are denied. Refer to [Force-update a worker pool with InPlace update strategy](shoot_operations.md#force-update-a-worker-pool-with-inplace-update-strategy) for more details.
+
+> ⚠️ Changing the update strategy from `AutoRollingUpdate` to `AutoInPlaceUpdate` or `ManualInPlaceUpdate` (and vice versa) is not allowed. However, switching between `AutoInPlaceUpdate` and `ManualInPlaceUpdate` is permitted.
+
+#### AutoInPlaceUpdate
+
+In case of AutoInPlaceUpdate update strategy, the update process is fully orchestrated by Gardener and the machine-controller-manager. No user intervention is required.The update strategy should be set to `AutoInPlaceUpdate` in the worker spec.
```

Comment:
```suggestion
In case of AutoInPlaceUpdate update strategy, the update process is fully orchestrated by Gardener and the machine-controller-manager. No user intervention is required. The update strategy should be set to `AutoInPlaceUpdate` in the Shoot spec under `.spec.provider.workers[].updateStrategy.
```